From 862c5a97a69d61ff9122072d8c7bb2e828086650 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Wed, 5 May 2021 16:05:17 +0200
Subject: [PATCH] fix(fips): add dh and ecdh ciphers

(cherry picked from commit 543b8014fc10fc6a92ba83db0dfc994fc1d2129b)

Resolves: #1775692
---
 modules.d/01fips/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/01fips/module-setup.sh b/modules.d/01fips/module-setup.sh
index f011f15d..8800a49e 100755
--- a/modules.d/01fips/module-setup.sh
+++ b/modules.d/01fips/module-setup.sh
@@ -24,7 +24,7 @@ installkernel() {
         _fipsmodules+="crc32c crct10dif ghash "
 
         # Ciphers:
-        _fipsmodules+="cipher_null des3_ede aes cfb "
+        _fipsmodules+="cipher_null des3_ede aes cfb dh ecdh "
 
         # Modes/templates:
         _fipsmodules+="ecb cbc ctr xts gcm ccm authenc hmac cmac "

